recipes/python-javabridge/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-javabridge" %}
+{% set version = "4.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9fd9c85e6a32f4edd9fff659d387cbf4c1c91fde9fe3670e4f6d37d79eb6200f
+
+build:
+  number: 0
+  script: '{{ PYTHON }} -m pip install --no-deps --ignore-installed . '
+  skip: true  # [win]
```

Comment:
Can you try generating the recipe with grayskull? It would fix things like the outdated script line here.